### DISCUSSION TITLE
Pot | 插件系统

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#668] [General]
# Pot | 插件系统

跨平台划词翻译和OCR软件

https://pot-app.com/docs/plugin.html

**Comment by neuralcraftsman** [comment]
能不能在国内的开源平台也发布一下

**Comment by elarbor** [comment]
[插件下载地址](https://pot-app.com/plugin.html#%E7%BF%BB%E8%AF%91)该放到界面上 
![image](https://github.com/user-attachments/assets/5dd8c7fe-00bf-4880-8df8-ecebd4a84b7b)
